tasks/section5.yml

```diff
@@ -51,15 +55,19 @@
       - rule_5.1.3
 
 - name: "SCORED | 5.1.3 | PATCH | Ensure permissions on /etc/cron.hourly are configured"
-  command: /bin/true
+  file:
+      dest=/etc/cron.hourly
+      owner=root
+      group=root
+      mode=0600
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md